### PR TITLE
Switch to pysys 2.0

### PR DESCRIPTION
**Pull request by abelikt**
## Proposed changes

- Switch to pysys 2.0
- update pip requirements
- Avoid project.username which seems to be used internally

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: tests/PySys/pysysproject.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<property name="tebasedir" value="${env.TEBASEDIR}" default="${testRootDir}/../.."/>
 	<property name="exampledir" value="${env.EXAMPLEDIR}" default="${testRootDir}/../../ci"/>
 	<property name="c8yurl" value="${env.C8YURL}" default=""/>
-	<property name="username" value="${env.C8YUSERNAME}" default=""/>
+	<property name="c8yusername" value="${env.C8YUSERNAME}" default=""/>
 	<property name="c8ypass" value="${env.C8YPASS}" default=""/>
 	<property name="tenant" value="${env.C8YTENANT}" default=""/>
 	<property name="deviceid" value="${env.C8YDEVICEID}" default=""/>
```
Comment (albinsuresh) [inline tests/PySys/pysysproject.xml:13]: While you're at renaming things, do you wanna rename this `deviceid` to `c8ydeviceid` as well? And may be rename `device` to `c8ydevicename`?

File: tests/PySys/pysysproject.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	<property name="tebasedir" value="${env.TEBASEDIR}" default="${testRootDir}/../.."/>
 	<property name="exampledir" value="${env.EXAMPLEDIR}" default="${testRootDir}/../../ci"/>
 	<property name="c8yurl" value="${env.C8YURL}" default=""/>
-	<property name="username" value="${env.C8YUSERNAME}" default=""/>
+	<property name="c8yusername" value="${env.C8YUSERNAME}" default=""/>
```
Comment (albinsuresh) [inline tests/PySys/pysysproject.xml:10]: I wish it was `c8y_username` but I see that you've just kept it consistent with the rest of the names.

**Review by albinsuresh — APPROVED** [review]

File: tests/PySys/pysysproject.xml (hunk 1 above)
Comment (abelikt) [reply tests/PySys/pysysproject.xml:13]: So far, it was just the bare minimum to get it working with the new pysys version. I would propose that we change the names with the next refactoring of the code. I've seen other things while browsing through the code.

File: tests/PySys/pysysproject.xml (hunk 2 above)
Comment (makr11st) [reply tests/PySys/pysysproject.xml:10]: There is always room for renaming all the vars.

File: tests/PySys/pysysproject.xml (hunk 1 above)
Comment (makr11st) [reply tests/PySys/pysysproject.xml:13]: Btw, why did you have to change? Is there some internal field with the same name?

**Pull request merged**